.github/workflows/cli-schedules-cron-schedule.yml

```diff
@@ -0,0 +1,33 @@
+name: cli-schedules-cron-schedule
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0/4 * * *"
+  pull_request:
+    branches:
+      - main
+      - sdk-preview
+    paths:
+      - cli/schedules/cron-schedule.yml
+      - .github/workflows/cli-schedules-cron-schedule.yml
+      - cli/setup.sh
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: check out repo
+      uses: actions/checkout@v2
+    - name: azure login
+      uses: azure/login@v1
+      with:
+        creds: ${{secrets.AZ_CREDS}}
+    - name: setup
+      run: bash setup.sh
```

Comment:
I think we manually created these workflows for schedule right?

Make sure this won't be overrided by the auto-gen script
cli/readme.py 
sdk/readme.py

Reply:
update: I added the content & workflow generate for cli/readme.py given there're 3 cli samples now. For SDK, I just manual update the readme.md

Reply:
And because schedule are new concept, the autogen script won't touch that part if we don't add new autogen code for schedule